karma.conf.js

```diff
@@ -77,11 +74,17 @@ module.exports = function(config) {
       noInfo: true,
     },
 
+    customLaunchers: {
+      Chrome_travis_ci: {
+        base: 'Chrome',
+        flags: ['--no-sandbox']
+      }
+    },
```

Comment:
I tried it, but Karma says no :(
```
09 09 2016 15:31:53.130:INFO [karma]: Karma v1.2.0 server started at http://localhost:8081/
09 09 2016 15:31:53.131:INFO [launcher]: Launching browser Headless_Chrome with unlimited concurrency
09 09 2016 15:31:53.138:INFO [launcher]: Starting browser Chrome
09 09 2016 15:32:53.137:WARN [launcher]: Chrome have not captured in 60000 ms, killing.
09 09 2016 15:32:53.227:INFO [launcher]: Trying to start Chrome again (1/2).
09 09 2016 15:33:53.231:WARN [launcher]: Chrome have not captured in 60000 ms, killing.
09 09 2016 15:33:53.371:INFO [launcher]: Trying to start Chrome again (2/2).
09 09 2016 15:34:53.369:WARN [launcher]: Chrome have not captured in 60000 ms, killing.
09 09 2016 15:34:53.468:ERROR [launcher]: Chrome failed 2 times (timeout). Giving up.
```
see https://github.com/karma-runner/karma-chrome-launcher/issues/10